pkg/querier/querier.go

```diff
@@ -40,6 +43,17 @@ type AgentNetworkPolicyInfoQuerier interface {
 	GetRuleByFlowID(ruleFlowID uint32) *types.PolicyRule
 }
 
+type AgentMulticastInfoQuerier interface {
+	// GetIGMPReportNPStats gets statistics generated by networkpolicies that block or allow
+	// IGMP reports message.
+	GetIGMPReportNPStats() map[cpv1beta.NetworkPolicyType]map[apitypes.UID]map[string]*types.RuleMetric
```

Comment:
Please add a comment saying the func will reset the saved statistics.

Reply:
updated